iconservice/icx/icx_engine.py

```diff
@@ -217,15 +219,14 @@ def get_balance(self,
         account: 'Account' = self._storage.get_account(context, address)
         return account.balance
 
-    # todo: need to be refactoring
-    def get_total_supply(self, context: 'IconScoreContext') -> int:
-        """Get the total supply of icx coin
+    def set_total_supply(self, total_supply: int):
```

Comment:
get_total_supply() -> total_supply_amount property
set_total_supply() -> not changed

Do you have any special reason?